packages/components/src/Modal/Modal.css

```diff
@@ -95,11 +95,7 @@
 }
 
 .closeButton {
-  padding: 0;
-  border: none;
-  background-color: transparent;
-  cursor: pointer;
-  appearance: none;
+  align-self: flex-start;
```

Comment:
This class wasn't used so I reused it for this

Reply:
Could you delete this class (and the wrapper in the TSX) and change the `align-items` [on the .header](https://github.com/GetJobber/atlantis/pull/1843/files#diff-155f6ca6b54ece4c9503ddeaa6a6d780bffb49359c5eb972356b1ce3a96da238R74) to `start`?

Reply:
ooooh I see why you can't, never mind me 😄